<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GaussianBeamWidget</class>
 <widget class="QWidget" name="GaussianBeamWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gaussian Beam</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout">
     <item>
      <widget class="QToolBox" name="toolBox">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>16</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_Bench">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>254</width>
          <height>479</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Optics bench</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Boundaries</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Left</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_LeftBoundary">
              <property name="suffix">
               <string> mm</string>
              </property>
              <property name="minimum">
               <double>-100000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>100000.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>10.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Right</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_RightBoundary">
              <property name="suffix">
               <string> mm</string>
              </property>
              <property name="minimum">
               <double>-100000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>100000.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>10.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_MagicWaist">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>228</width>
          <height>360</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Magic waist</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Aspect</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_TargetOrientation">
           <item>
            <property name="text">
             <string>Spherical</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ellipsoidal</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Target waist</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QDoubleSpinBox" name="doubleSpinBox_HTargetWaist">
             <property name="maximum">
              <double>100000.000000000000000</double>
             </property>
             <property name="value">
              <double>100.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="doubleSpinBox_VTargetWaist">
             <property name="maximum">
              <double>100000.000000000000000</double>
             </property>
             <property name="value">
              <double>100.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Target position</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QDoubleSpinBox" name="doubleSpinBox_HTargetPosition">
             <property name="minimum">
              <double>-100000.000000000000000</double>
             </property>
             <property name="maximum">
              <double>100000.000000000000000</double>
             </property>
             <property name="value">
              <double>550.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="doubleSpinBox_VTargetPosition">
             <property name="minimum">
              <double>-100000.000000000000000</double>
             </property>
             <property name="maximum">
              <double>100000.000000000000000</double>
             </property>
             <property name="value">
              <double>550.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_MinOverlap">
           <property name="text">
            <string>Minimum overlap</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_MinOverlap">
           <property name="suffix">
            <string> %</string>
           </property>
           <property name="maximum">
            <double>100.000000000000000</double>
           </property>
           <property name="value">
            <double>98.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_ShowTargetBeam">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Show target beam</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_OverlapResult">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_MagicWaistResult">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_LocalOptimum">
           <property name="text">
            <string>Local optimum</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_MagicWaist">
           <property name="text">
            <string>Find</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_Fit">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>237</width>
          <height>511</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Waist fit</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QComboBox" name="comboBox_Fit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_AddFit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>Add a new fit</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="GaussianBeam.qrc">
               <normaloff>:/images/edit-add.png</normaloff>:/images/edit-add.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_RemoveFit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>Delete this fit</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="GaussianBeam.qrc">
               <normaloff>:/images/edit-delete.png</normaloff>:/images/edit-delete.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>color</string>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
             <property name="margin">
              <number>1</number>
             </property>
             <property name="openExternalLinks">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_FitColor">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_RenameFit">
             <property name="toolTip">
              <string>Rename the current fit</string>
             </property>
             <property name="text">
              <string>Rename</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Orientation:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_FitOrientation">
           <item>
            <property name="text">
             <string>Spherical</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Horizontal</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Vertical</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ellipsoidal</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Measured quantity:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_FitData">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>Radius @ 1/e²</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Diameter @ 1/e²</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Standard deviation</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FWHM</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>HWHM</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Measurements:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableView" name="fitTable"/>
         </item>
         <item>
          <layout class="QHBoxLayout">
           <item>
            <widget class="QPushButton" name="pushButton_FitAddRow">
             <property name="text">
              <string>Row</string>
             </property>
             <property name="icon">
              <iconset resource="GaussianBeam.qrc">
               <normaloff>:/images/edit-add.png</normaloff>:/images/edit-add.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_FitRemoveRow">
             <property name="text">
              <string>Row</string>
             </property>
             <property name="icon">
              <iconset resource="GaussianBeam.qrc">
               <normaloff>:/images/edit-delete.png</normaloff>:/images/edit-delete.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_FitResult">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_SetInputBeam">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Set as input beam</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_SetTargetBeam">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Set as target beam</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="GaussianBeam.qrc"/>
 </resources>
 <connections/>
</ui>
